### PR TITLE
[ZEPPELIN-5078] Make it possible to cancel "run all paragraphs" operation

### PR DESCRIPTION
**Pull request by olkkoti**
### What is this PR for?

Make it possible to cancel "run all paragraphs" operation. Currently, Cancel is blocked by UI as well as blocking "run all" operation on server. This PR will unblock cancel from UI and do "run all" operation in separate thread and so avoid blocking the websocket.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-5078

### How should this be tested?
Two automatic tests in zeppelin-integration package.
I have problems with CI, but I am able to run them successfully locally.

### CODE REVIEW

File: flink-cmd/pom.xml [hunk 1]
```diff
@@ -23,14 +23,14 @@
   <parent>
     <artifactId>zeppelin-interpreter-parent</artifactId>
     <groupId>org.apache.zeppelin</groupId>
-    <version>0.10.0-SNAPSHOT</version>
+    <version>0.9.1-SNAPSHOT</version>
```
Comment (zjffdu) [inline flink-cmd/pom.xml:26]: This change is not necessary. We use 0.9.1-SNAPSHOT for branch-0.9, and 0.10.0-SNAPSHOT for master

Comment (olkkoti) [reply flink-cmd/pom.xml:26]: Ah, should parents refer to version 0.10.0 in both flink-cmd and spark-submit? At least I am not able to build that way.

Comment (zjffdu) [reply flink-cmd/pom.xml:26]: No, all the versions should be 0.9.1-SNAPSHOT, do you mean you unable to build branch-0.9 ?

Comment (olkkoti) [reply flink-cmd/pom.xml:26]: Yes, I am building on branch-0.9. Should I target this bug fix to master instead?

Comment (zjffdu) [reply flink-cmd/pom.xml:26]: Yes, usually we create PR based on master branch and will cherry-pick that into branch-0.9 when merging.

Comment (olkkoti) [reply flink-cmd/pom.xml]: Ok, I changed to master.

**Comment by zjffdu** [comment]
@olkkoti There're still some failed selenium tests. Although there're some flaky test in selenium tests, but not sure whether it is related with your PR. Could you help to take a look at that ?

**Comment by olkkoti** [comment]
> @olkkoti There're still some failed selenium tests. Although there're some flaky test in selenium tests, but not sure whether it is related with your PR. Could you help to take a look at that ?

I fixed my cancel tests to have shorter sleep times in paragraphs, and made so that they actually complete even if the tests fail.
The latest run seems to be a lot better, however there is still one error.

**Comment by zjffdu** [comment]
@olkkoti I think this is unrelated, because I see this error in master as well. But it would better to run CI again, because this is flaky test, sometimes fails and sometimes succeed.

**Comment by zjffdu** [comment]
Thanks @olkkoti  CI is passed

**Comment by zjffdu** [comment]
Will merge if no more comment

**Pull request closed without merging**